src/components/ShareDialog.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  Dialog,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+} from '@material-ui/core';
+import {
+  FacebookShareButton,
+  FacebookIcon,
+  EmailShareButton,
+  EmailIcon,
+  LinkedinShareButton,
+  LinkedinIcon,
+  RedditShareButton,
+  RedditIcon,
+  TwitterShareButton,
+  TwitterIcon,
+} from 'react-share';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    background: theme.palette.background.secondary,
+    color: 'white',
+    fontSize: '15px',
+    textAlign: 'center',
+  },
+  shareIcon: {
+    maringRight: '15px',
```

Comment:
Use `theme.padding(N)` (where _N_ is a number) to set this.

Reply:
Wait did you mean -> `theme.spacing(N)` ?

Reply:
Yeah that's what I meant, sorry.